simulation.py

```diff
@@ -74,21 +74,21 @@ def __init__(self, *args, **kwargs):
         # create placeholders for the plots MainWindow will delivering (updating)
         # data into.
         self.graphs = pg.GraphicsLayoutWidget(show=True, title="Race Sim plots")
-        self.graphs.resize(1000,540)
-        self.p1 = self.graphs.addPlot(name="Plot1", title="Time (s)")        
```

Comment:
It looks like quite a few file modifications are due to differences in new-line, end-of-line and spaces. Perhaps we have our editors set up differently and should agree on some standard settings.